src/pageTypes/components/PageTypeBulkDeleteDialog/PageTypeBulkDeleteDialog.tsx

```diff
@@ -29,27 +29,27 @@ const PageTypeBulkDeleteDialog: React.FC<PageTypeBulkDeleteDialogProps> = ({
       onClose={onClose}
       onConfirm={onConfirm}
       title={intl.formatMessage({
-        id: "CcEwXH",
-        defaultMessage: "Delete Page Types",
+        id: "/jU+sI",
+        defaultMessage: "Delete model types",
         description: "dialog header",
       })}
       variant="delete"
     >
       {hasPages ? (
         <FormattedMessage
-          id="2Zyit2"
-          defaultMessage="{counter,plural,one{Page Type you want to delete is used by some pages. Deleting this page type will also delete those pages. Are you sure you want to delete this page type? After doing so you won’t be able to revert changes.} other{Page Types you want to delete are used by some pages. Deleting these page types will also delete those pages. Are you sure you want to delete {displayQuantity} page types? After doing so you won’t be able to revert changes.}}"
-          description="delete page types with its pages"
+          id="45W13E"
+          defaultMessage="{counter,plural,one{Model type you want to delete is used by some models. Deleting this model type will also delete those models. Are you sure you want to delete this model type? After doing so you won’t be able to revert changes.} other{Model types you want to delete are used by some models. Deleting these model types will also delete those models. Are you sure you want to delete {displayQuantity} model types? After doing so you won’t be able to revert changes.}}"
```

Comment:
question: do we want to use `’` as an apostrophe instead of `'`? I think we are using `'`' everywhere, so maybe we should stick with that one?

Reply:
hmm good question i think we can standardise it later on + add some spell checking